### DISCUSSION TITLE
This application requires a Java Runtime Environment 17.

### DISCUSSION THREAD

**Discussion by nottheman686** [mircokroon/minecraft-world-downloader#570] [General]
Every time i try to use the exe i get this error message and it opens a page on java. Im not familiar with how to get what the error is requesting.

**Comment by Petrosmine** [comment]
micro man pls help there is no download link for the java thing give us link

**Comment by foxnadir** [comment]
It is not Runtime but Java Development Kit version. Here is official link:
https://www.oracle.com/cis/java/technologies/downloads/#jdk20-windows

**Comment by nottheman686** [comment]
Thank you friend. This is going to be super useful. Have a project I was
starting to get burnt out on because of how long it's taken me to work on
it. Now I can try out build ideas a little quicker
